.ci/scripts/linux-flatpak/finish.sh

```
@@ -5,5 +5,5 @@ REPO_DIR="$YUZU_SRC_DIR/repo"
 
 # When the script finishes, unmount the repository and delete sensitive files,
 # regardless of whether the build passes or fails
-#umount "$REPO_DIR" || true
-#rm -rf "$REPO_DIR" "/tmp/*" || true
+umount "$REPO_DIR" || true
+rm -rf "$REPO_DIR" "/tmp/*" || true
```

Change this to

```bash
rm -r "$REPO_DIR" /tmp/* || true
```

It's no use to hide errors from log and quoting `/tmp/*` makes it fail. Try this out locally to see:

```bash
mkdir test
touch test/{1..3}
rm -r "test/*"
```

Result is `rm: cannot remove 'test/*': No such file or directory`.